extractor/src/main/kotlin/de/snowii/extractor/extractors/Blocks.kt

```diff
@@ -52,10 +52,14 @@ class Blocks : Extractor.Extractor {
             for (state in block.stateManager.states) {
                 val stateJson = JsonObject()
                 val id = stateIdCounter
+
                 stateIdCounter++
+
                 stateJson.addProperty("id", id)
                 stateJson.addProperty("luminance", state.luminance)
+                stateJson.addProperty("opacity", state.opacity)
```

Comment:
I would put this below the opaque boolean

Reply:
Actually we can may remove the opaque and only write opacity when the block has opaque. So in rust we could make it an Option<Value>